src/main/scala/com/microsoft/partnercatalyst/fortis/spark/StreamProviderFactory.scala

```diff
@@ -0,0 +1,43 @@
+package com.microsoft.partnercatalyst.fortis.spark
+
+import java.io.File
+
+import com.microsoft.partnercatalyst.fortis.spark.streamfactories.adapters.TadawebAdapter
+import com.microsoft.partnercatalyst.fortis.spark.streamfactories._
```

Comment:
In general, I agree. In this case, I don't see the harm. We control the contents of the stream factories package, and we'll need to instantiate all of those factories within this class.